libs/langchain-google-common/src/tests/llms.test.ts

```diff
@@ -521,4 +521,30 @@ describe("Mock Google LLM", () => {
     expect(responseArray).toHaveLength(3);
     console.log("record", JSON.stringify(record, null, 2));
   });
+
+  test("8: streamGenerateContent - streaming - json responseMimeType", async () => {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const record: Record<string, any> = {};
+    const projectId = mockId();
+    const authOptions: MockClientAuthInfo = {
+      record,
+      projectId,
+      resultFile: "llm-8-mock.json",
```

Comment:
I don't see the llm-8-mock.json file checked in.

Reply:
my bad, will push it asap